### PR TITLE
OTJ 10 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/quick_draw.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Name:Quick Draw
+ManaCost:R
+Types:Instant
+A:SP$ Pump | NumAtt$ +1 | NumDef$ +1 | KW$ First Strike | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | SubAbility$ DBPump | SpellDescription$ Target creature you control gets +1/+1 and gains first strike until end of turn.
+SVar:DBPump:DB$ Pump | ValidTgts$ Opponent | SubAbility$ DBCurse
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/quick_draw.txt:5]: extra pump unneeded

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/quick_draw.txt]: you still need to target the opponent though

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/rakdos_joins_up.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Rakdos Joins Up
+ManaCost:3 B R
+Types:Legendary Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChange | TriggerDescription$ When CARDNAME enters the battlefield, return target creature card from your graveyard to the battlefield with two additional +1/+1 counters on it.
+SVar:TrigChange:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Creature.YouOwn | WithCountersType$ P1P1 | WithCountersAmount$ 2 | TgtPrompt$ Select target creature card in your graveyard
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.Legendary+YouCtrl | Execute$ TrigDamage | TriggerDescription$ Whenever a legendary creature you control dies, CARDNAME deals damage equal to that creature's power to target opponent.
+SVar:TrigDamage:DB$ DealDamage | ValidTgts$ Opponent | NumDmg$ Damage
+SVar:Damage:TriggeredCard$CardPower
+DeckHas:Ability$Graveyard|Counters
+DeckHints:Ability$Graveyard|Mill & Type$Legendary
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/rakdos_joins_up.txt:10]: maybe add Discard|Sacrifice to AI abilities

File: forge-gui/res/cardsfolder/upcoming/rakish_crew.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Rakish Crew
+ManaCost:2 B
+Types:Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a 1/1 red Mercenary creature token with "{T}: Target creature you control gets +1/+0 until end of turn. Activate only as a sorcery."
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ r_1_1_mercenary_tappump | TokenOwner$ You
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Outlaw | Execute$ TrigDrain | TriggerDescription$ Whenever an outlaw you control dies, each opponent loses 1 life and you gain 1 life. (Assassins, Mercenaries, Pirates, Rogues, and Warlocks are outlaws.)
+SVar:TrigDrain:DB$ LoseLife | Defined$ Player.Opponent | LifeAmount$ 1 | SubAbility$ DBGainLife
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/rakish_crew.txt:7]: might as well `Defined$ Opponent`

**Review by tool4ever — APPROVED** [review]

**Pull request merged**